packages/tc-api-rest-handlers/__tests__/get.spec.js

```diff
@@ -6,37 +6,36 @@ describe('rest GET', () => {
 	const namespace = 'api-rest-handlers-get';
 	const mainCode = `${namespace}-main`;
 	const input = {
-		type: 'MainType',
+		type: 'SimpleGraphBranch',
 		code: mainCode,
 	};
 
-	const { createNodes, createNode, connectNodes, meta } = setupMocks(
+	const { createNodes, createNode, connectNodes, stockMetadata } = setupMocks(
 		namespace,
 	);
 
-	const createMainNode = (props = {}) =>
```

Comment:
Highlight #5 Most of the clever helpers are gone. Left behind are some generic things like `createNode()` and `connectNodes()` but almost all wrapped version of these are gone in favour of actually just calling the generic with the actual full set of arguments in each test - slightly more verbose, but more comprehensible